### ISSUE TITLE
Can I change the value of RadarChart dynamically?

### ISSUE THREAD

**Issue by yunomiyako** [foolsong/EasyChartsSwift#1]
I want Radar Chart to change dynamically so I implemented sliders under the radar chart.
And the radar chart changed by the value of sliders.
Every time that the value of sliders change, the below code is called for drawing radar chart.
`radarChartView.setupRadarChartConfig(radarChartConfig: radarChartConfig)`

However, It becomes more and more heavy.
How can I solve this?
Or do I have to implement a button like "UPDATE", and only time when it is pushed the above code is called?

**Comment by foolsong** [comment]
Yes, of course, but recently I was busy with work and I could update my method over the weekend.
